### ISSUE TITLE
Rule proposal: Warn on duplicate props

### ISSUE THREAD

**Issue by hummlas** [yannickcr/eslint-plugin-react#143]
When you're in strict-mode you get warnings when you have duplicate properties in an object, a similiar warning for props on React components would be nice.

```javascript
return (
   <Mycomponent className={classes}
                            onClick={this.handleClick}
                            someOtherProp={stuff}
                            onClick={this.onClick} /> // Duplicate prop, should produce an ESLint error
);
```

**Comment by mathieumg** [comment]
:+1: +1
